content/docs/1.13/performance-tuning.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Performance Tuning Guide
+description: Tweaking your Jaeger instance to achieve a better performance
+weight: 10
+---
+
+Jaeger was built from day 1 to be able to ingest huge amounts of data in a resilient way. To better utilize resources that might cause delays, such as storage or network communications, Jaeger buffers and batches data. When more spans are generated than Jaeger is able to safely process, spans might get dropped. However, the defaults might not fit all scenarios: for instance, agents running as a sidecar might have more memory constraints than agents running as a daemon in bare metal.
+
+## Adjusting the client settings
```

Comment:
For consistency, perhaps use these titles:
  * Client (tracer) settings
  * Agent settings
  * Collector settings (current title)

Reply:
Usually I'd say yes, but in this particular case I think we're OK with having the Level 1 headings that Yuri recommends (the architectural components), as long as we have verbs with the Level 2 headings (which is where you're suggesting some sort of action).
So maybe something like this?  (Feel free to pick different verbs, this is just a quick example)
Client (Tracer) settings
-Adjust sampling configuration
-Increase in-memory queue size
- Modify batched spans flush interval
Agent Settings
- Increase server queue sizes
- Increase processor workers
Collector Settings
- Increase queue size
- Increase workers

Reply:
I changed the doc to use your suggestions, except for the last one, which I used "Increase processor workers" (like in the Agent settings).